### PR TITLE
1.16.5 Update

### PR DESCRIPTION
**Pull request by X-Niter**
Please take a detailed look into the changes and let me know if you feel something is off. 

There is not anything wrong with this update on a normal user standpoint, but in the coders eyes I noticed a method of making the players uuid the characters uuid didn't really do that.. From running on the current public working version I noticed the same method also didn't actually do this, please let me know any information as to how that's supposed to function as I don't think it ever "truelly" applied the uuid spoof. 
I know on another Repo I have, I've manage a full uuid spoof how ever that was with using Bungeecord methods. 

Anyways, other than that small bit, everything else here works and was super fun to test out, I was very tempted to add more to this plugin such as "More slots", and changing the onDisable function as getting kicked that way on a normal server restart could result in the Server owner/admin and players to be making questions of concerns as to why there getting "Kicked" with a corruption message instead of the restart message.

#3 Still Happens unfortunately, but only if I'm the one purposely being in a super big rush and pressing to many buttons and stuff, all I can say for this one, is don't be in such a hurry and don't spamm buttons. But I'll keep a look out on this and see if there "might" be a resolution on it. 

#5 This is the 1.16.5 update, maybe closed after this PR is accepted.

#4 I did not really look into this, but it wouldn't be hard to go through the whole plugin and make sure all messages are in the message.yml

#2 I took this into thought, and I tested this plugin with MultiVerse and my base world being a MultiVerse world, although I did add into the code that if the world fails it will fall back to default world 0 in the case the config was not configured correctly.

Hope this brings some happiness to some folks! :)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -163,7 +163,7 @@
                             <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <skip>false</skip>
+                            <skip>true</skip>
```
Comment (Foxxite) [inline pom.xml:166]: This bat files copies the compiled jar to my dev server. You can find it in the scripts folder.

**Review by Foxxite — APPROVED** [review]

**Review by Foxxite — COMMENTED** [review]
You make some things more verbose, and other less verbose. Might be wise to be more consistent.

**Review by Foxxite — APPROVED** [review]

**Review by Foxxite — APPROVED** [review]

**Comment by Foxxite** [comment]
Once I have verified that this works without breaking configs from the current public build, I will merge. Will probably do this somewhere this week. If I forget, reply here.

**Comment by X-Niter** [comment]
> You make some things more verbose, and other less verbose. Might be wise to be more consistent.
- I'll keep that in mind, thank you for the feedback, I do apologize if it was not up to par, my biggest goal in mind set was to change as little as possible to achieve the appropriate outcome, there however was many things that could have gone without doing for sure.

>Once I have verified that this works without breaking configs from the current public build, I will merge. Will probably do this somewhere this week. If I forget, reply here.

- Sounds good, I'll touch base the end of the week/weekend then. 
I still am slightly curious how the full "Player to Character UUID" is supposed to be handled, that may just be my eyes missing a small detail, however.

**Pull request merged**